libraries/WiFi/src/WiFiGeneric.h

```diff
@@ -31,6 +31,12 @@
 #include "esp_smartconfig.h"
 #include "wifi_provisioning/manager.h"
 
+#ifdef ENABLE_WPA2_AUTHENTICATION
+#include <WiFiClient.h>
+#include <WiFiClientSecure.h>
```

Comment:
why are those two included? Just `esp_wpa2.h` is enough, no?

Reply:
I don't think that esp_wpa2.h is enough. If you look just above the new WiFiSTAClass::begin() function that I created, I declared a static WiFiClientSecure. This is necessary for negotiating the various types of cert processes that are sometimes needed with WPA2 Enterprise. esp_wpa2.h does not include WiFiClientSecure.h, as the former is a esp-idf header and the latter is an arduino-espressif32 header.

I could/should arguably move these into the WiFiSTA.cpp file (and I don't think I actually need WiFiClient.h) to prevent unnecessary #includes, however.

Reply:
yes please :)

Reply:
MSVALIDATE.01 ¤ "Meta" <tag>
-----[\HTTPS://BITCOINNEWS.COM----]

Reply:
What is was is s someone added their Business camera sytem to my computer... now i have identity issues

Reply:
"Screwballs"tied his camera system to my computer... now i cannot access any of my own stuff
![20220405_174156](https://user-images.githubusercontent.com/103056448/161875533-5f234cb4-cb9d-4b62-9255-347a0a60323e.jpg)